### PR TITLE
KGPE D16 test matrix

### PR DESCRIPTION
**Pull request by sulewskiprzemyslaw**
Signed-off-by: Przemysław Sulewski <przemyslaw.sulewski@3mdeb.com>

### CODE REVIEW

**Review by damiankaras — APPROVED** [review]
I don't have points.

File: docs/unified-test-documentation/test-matrix.md [hunk 1]
```diff
@@ -29,6 +31,12 @@ described in this documentation are dedicated to the given platform
 | 1.   | TPM Support                                       | TPM           | TPM001.001                           |
 | 2.   | Verified Boot support                             | VBO           | VBO001.001, VBO002.001, VBO003.001   |
 
+### Module: Dasharo performance
+
+| No.  | Supported test suite                              | Test suite ID | Supported test cases                 |
+|:-----|:--------------------------------------------------|:-------------:|:-------------------------------------|
+| 1.   | Coreboot boot measure                             | CBMEM         | All                                  |
```
Comment (pietrushnic) [inline docs/unified-test-documentation/test-matrix.md:38]:
```suggestion
| 1.   | coreboot boot measure                             | CBMEM         | All                                  |
```
@macpijan can we have pre-commit for such corrections?

Comment (sulewskiprzemyslaw) [reply docs/unified-test-documentation/test-matrix.md]: All occurrences of the word `coreboot` have been improved in test-matrix file.

File: docs/unified-test-documentation/test-matrix.md [hunk 2]
```diff
@@ -29,6 +31,12 @@ described in this documentation are dedicated to the given platform
 | 1.   | TPM Support                                       | TPM           | TPM001.001                           |
 | 2.   | Verified Boot support                             | VBO           | VBO001.001, VBO002.001, VBO003.001   |
 
+### Module: Dasharo performance
+
+| No.  | Supported test suite                              | Test suite ID | Supported test cases                 |
+|:-----|:--------------------------------------------------|:-------------:|:-------------------------------------|
+| 1.   | coreboot boot measure                             | CBMEM         | All                                  |
```
Comment (miczyg1) [inline docs/unified-test-documentation/test-matrix.md:38]: This test case needs better name, in PCE we have coreboot booting time. Maybe `coreboot bring up time measurement` ?

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: docs/unified-test-documentation/test-matrix.md (hunk 2 above)
Comment (sulewskiprzemyslaw) [reply docs/unified-test-documentation/test-matrix.md]: Okay, I will make the change.

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**